patches/tModLoader/Terraria.ModLoader/GlobalItem.cs

```diff
@@ -479,14 +479,28 @@ public virtual void OpenVanillaBag(string context, Player player, int arg)
 		}
 
 		/// <summary>
-		/// This hooks gets called immediately before an item gets reforged by the Goblin Tinkerer. Useful for storing custom data, since reforging erases custom data.
+		/// Returns if the normal reforge pricing is applied. 
+		/// If true or false is returned and the price is altered, the price will equal the altered price.
+		/// The passed reforge price equals the item.value. Vanilla pricing will apply 20% discount if applicable and then price the reforge at a third of that value.
 		/// </summary>
-		public virtual void PreReforge(Item item)
```

Comment:
Do we know of any mods using PreReforge? Making a breaking change like this might be OK if no mods actually ever got around to using it. Otherwise we need a `bool NewPreReforge` that by default calls PreReforge, like what we did with when we had to change the Wing hooks. Also mark PreReforge as Obsolete until 0.11.

Reply:
Woops, you're right. I didn't think about that. For sure there are plenty of mods using it because it had to be used to preserve modded data.